tests/framework/installer/ceph_csi_templates.go

```diff
@@ -332,6 +332,21 @@ const (
               volumeMounts:
                 - name: socket-dir
                   mountPath: /csi
+            - name: csi-snapshotter
+              image:  {{ .SnapshotterImage }}
+              args:
+                - "--csi-address=$(ADDRESS)"
+                - "--v=5"
+                - "--timeout=60s"
```

Comment:
why the 60s? leader election flags are missing here

Reply:
Increased to 150. also added leader election for both cephfs and RBD as it was missing. :+1: